Automattic/wp-calypso
# Simple Payments: Remove feature flag

Fix #25326

Remove the `simple-payments` feature flag from all environments, and all its uses across Calypso.

Also, take care of some a11y concerns in the HTML Editor toolbar's Add Content dropdown: replace some `div` with click events with `button` so that the commit does not blow up several ESLint's `jsx/a11y` rules.

## Testing instructions

- Make sure that the Simple Payments dialog still works in both Visual and HTML editor for Premium and Business sites.
- Make sure Simple Payments are not enabled for Free and Personal sites.
- Additionally: check that the Add Content dropdown in the HTML toolbar still works the same as the one in the Visual toolbar.
